chatkit-core/src/main/kotlin/com/pusher/chatkit/messages/MessageService.kt

```diff
@@ -1,78 +1,70 @@
 
 internal class MessageService(
     private val legacyV2client: PlatformClient,
     private val client: PlatformClient,
     private val userService: UserService,
     private val roomService: RoomService,
-    private val urlRefresher: UrlRefresher,
+//    private val urlRefresher: UrlRefresher,
     private val filesService: FilesService
 ) {
+    @Suppress("UNUSED_PARAMETER")
     fun fetchMessages(
         roomId: String,
         limit: Int,
         initialId: Int?,
         direction: Direction
-    ): Result<List<Message>, Error> =
-            fetchMessagesParams(limit, initialId, direction).let { params ->
-                legacyV2client.doGet<List<Message>>("/rooms/$roomId/messages$params").flatMap { messages ->
-                    messages.map { message ->
-                        userService.fetchUserBy(message.userId).map { user ->
-                            message.user = user
-                            message
-                        }
-                    }.collect().mapFailure { errors ->
-                        Errors.compose(errors)
-                    }
-                }
-            }
+    ) {}
+//            : Result<List<Message>, Error> =
+//            fetchMessagesParams(limit, initialId, direction).let { params ->
+//                legacyV2client.doGet<List<Message>>("/rooms/$roomId/messages$params").flatMap { messages ->
+//                    messages.map { message ->
+//                        userService.fetchUserBy(message.userId).map { user ->
+//                            message.user = user
+//                            message
+//                        }
+//                    }.collect().mapFailure { errors ->
+//                        Errors.compose(errors)
+//                    }
+//                }
+//            }
 
+    @Suppress("UNUSED_PARAMETER")
     fun fetchMultipartMessages(
         roomId: String,
         limit: Int,
         initialId: Int?,
         direction: Direction
-    ): Result<List<com.pusher.chatkit.messages.multipart.Message>, Error> =
-            fetchMessagesParams(limit, initialId, direction).let { params ->
-                client.doGet<List<V3MessageBody>>("/rooms/$roomId/messages$params").flatMap { messages ->
-                    messages.map { message ->
-                        upgradeMessageV3(
-                                message,
-                                roomService,
-                                userService,
-                                urlRefresher
-                        )
-                    }.collect().mapFailure { errors ->
-                        Errors.compose(errors)
-                    }
-                }
-            }
+    ) {}
+//            : Result<List<com.pusher.chatkit.messages.multipart.Message>, Error> =
+//            fetchMessagesParams(limit, initialId, direction).let { params ->
+//                client.doGet<List<V3MessageBody>>("/rooms/$roomId/messages$params").flatMap { messages ->
```

Comment:
since `V3MessageBody` has been renamed it would be good to keep up-to-date name in the commented out code too

Reply:
I see your point - i don't think the refactor tool looks at commented out code. As some of the code depends on methods that no longer exist e.g. ` upgradeMessageV3(` I figure as soon as someone uncomments the block (if they need it) it'll be easy to update with the intended object?

Reply:
oh yes true, unfortunately rename refactoring doesn't look at the comments. Now I am thinking that since it will be hard to work from such commented out code, that it would be better to remove it. It will be better use the compiling version in the `master` to bring such logic to v2. Also we won't need to remember to clean up commented out code later. Which may be easy to forget when the new code lands in a different place (even different part of the same file). What are your thoughts?

Reply:
Yeah, think I agree. We have the history on master so if people want to check they can check there. I've deleted the classes if they had lots of sections commented out, or weren't currently being used by anything :)